waku/v2/node/storage/message/message_store.nim

```diff
@@ -31,38 +30,11 @@ type
 
   MessageStore* = ref object of RootObj
 
-# TODO: Deprecate the following type
-type DataProc* = proc(receiverTimestamp: Timestamp, msg: WakuMessage, pubsubTopic: string) {.closure, raises: [Defect].}
-
-
-# TODO: Remove after resolving nwaku #1026. Move it back to waku_store_queue.nim
-type 
-  IndexedWakuMessage* = object
-    # TODO may need to rename this object as it holds both the index and the pubsub topic of a waku message
-    ## This type is used to encapsulate a WakuMessage and its Index
-    msg*: WakuMessage
-    index*: Index
-    pubsubTopic*: string
-    
-  QueryFilterMatcher* = proc(indexedWakuMsg: IndexedWakuMessage) : bool {.gcsafe, closure.}
-
 
 # MessageStore interface
-method getMostRecentMessageTimestamp*(db: MessageStore): MessageStoreResult[Timestamp] {.base.} = discard
-
-method getOldestMessageTimestamp*(db: MessageStore): MessageStoreResult[Timestamp] {.base.} = discard
-
 method put*(db: MessageStore, cursor: Index, message: WakuMessage, pubsubTopic: string): MessageStoreResult[void] {.base.} = discard
```

Comment:
Perhaps change `db` to just `ms` in the base methods? Makes it a bit clearer that implementations of `MessageStore` are now not necessarily a database.

Reply:
Agree. I'll change it 😁